### PR TITLE
[IMP] web: add margin for statusbar_buttons icons 

### PR DESCRIPTION
**Pull request by dbeguin**
Before this commit, when adding a icon to a status bar button, there was no
margin between icon and text.

After this commit, for readability purpose, a margin has been added to the
icons of a status bar button.

Task ID: 2083049

### CODE REVIEW

File: addons/mail/models/mail_tracking_value.py [hunk 1]
```diff
@@ -39,7 +39,9 @@ def _compute_field_groups(self):
         for tracking in self:
             model = self.env[tracking.mail_message_id.model]
             field = model._fields.get(tracking.field.name)
-            tracking.field_groups = field.groups
+            # The field could not exists anymore, for example when
+            # calling message_change_thread
+            tracking.field_groups = field.groups if field else 'base.group_system'
```
Comment (awa-odoo) [inline addons/mail/models/mail_tracking_value.py:44]: @dbeguin Not sure what to do with this fix.

Does this apply to prior stable versions?
Could you add a unit test to demonstrate the issue and also ensure this fixes it?

Thanks :)

File: addons/crm/views/crm_lead_views.xml [hunk 2]
```diff
@@ -24,6 +24,7 @@
                     </header>
                     <sheet>
                         <field name="active" invisible="1"/>
+                        <widget name="web_ribbon" title="Archived" bg_color="bg-danger" attrs="{'invisible': [('active', '=', True)]}"/>
```
Comment (tde-banana-odoo) [inline addons/crm/views/crm_lead_views.xml:27]: There is a "Lost" ribbon that may conflict with this one (active=False, probability 0 -> two ribbons normally).

File: addons/mail/models/mail_tracking_value.py (hunk 1 above)
Comment (tde-banana-odoo) [reply addons/mail/models/mail_tracking_value.py:44]: +1 a test would be welcomed :)

Comment (dbeguin) [reply addons/mail/models/mail_tracking_value.py:42]: Well that was already fixed but the fix was stuck in 13.0.
I took here the 'manual FP recovering' commit, that have been made in 13.2, to make the ENT branch working while testing the feature.

File: addons/web/static/src/scss/form_view.scss [hunk 3]
```diff
@@ -191,6 +191,10 @@
                 margin: $o-statusbar-buttons-vmargin 3px $o-statusbar-buttons-vmargin 0;
                 padding-top: 2px;
                 padding-bottom: 2px;
+
+                > .o_button_icon {
+                    margin-right: $o-statbutton-spacing;
```
Comment (dbeguin) [inline addons/web/static/src/scss/form_view.scss:196]: could be 4px. meh'.. this way we are aligned with spacing with other button icons. but this variable is not really related with statusbar_buttons.. :shrug:

**Comment by tde-banana-odoo** [comment]
@robodoo r+ rebase-ff
